packages/coordinator/src/screens/scheduledAppointments/ScheduledAppointmentsScreenContainer.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import {
+  BookedDonationWithDonorDetails,
+  DateUtils,
+  Hospital,
+  HospitalUtils,
+} from "@zm-blood-components/common";
+import * as CoordinatorFunctions from "../../firebase/CoordinatorFunctions";
+import ScheduledAppointmentsScreen from "./ScheduledAppointmentsScreen";
+import HeaderSection from "../../components/HeaderSection";
+import Select from "../../components/Select";
+import DatePicker from "../../components/DatePicker";
+import Button from "../../components/Button";
+import styles from "./ScheduledAppointmentsScreen.module.scss";
+import CsvDownloaderButton from "../../components/CsvDownloaderButton";
+import {
+  csvColumns,
+  formatDataByColumns,
+} from "./ScheduledAppointmentsCsvConfig";
+
+export default function ScheduledAppointmentsScreenContainer() {
+  const TWO_WEEKS_IN_MILLIS = 1000 * 60 * 60 * 24 * 14;
+  const initialToDate = new Date(Date.now() + TWO_WEEKS_IN_MILLIS);
+
+  const [fromDate, setFromDate] = useState<Date | null>(new Date());
+  const [toDate, setToDate] = useState<Date | null>(initialToDate);
+  const [hospital, setHospital] = useState<Hospital | "">("");
+  const [isLoading, setIsLoading] = useState(false);
+  const [appointmentsWithDonorDetails, setAppointmentsWithDonorDetails] =
+    useState<BookedDonationWithDonorDetails[]>([]);
+
+  const onSearch = async () => {
+    if (!hospital || !fromDate || !toDate) {
+      return;
+    }
+    setIsLoading(true);
+    // get all bookings in fromDate day
+    const fromDateMidnight = DateUtils.DateToMidnight(fromDate);
+    const nextAppointmentsWithDonorDetails =
+      await CoordinatorFunctions.getBookedAppointmentsInHospital({
+        hospital,
+        fromDateMillis: fromDateMidnight!.getTime(),
+        toDateMillis: toDate!.getTime(),
+      });
+    setAppointmentsWithDonorDetails(nextAppointmentsWithDonorDetails);
+    setIsLoading(false);
+  };
+
+  return (
+    <>
+      <HeaderSection className={styles.component}>
+        <Select
+          id={"hospital"}
+          label={"בית חולים"}
+          options={HospitalUtils.getAllHospitalOptions()}
```

Comment:
I added an ability to show only hospitals relevant to coordinator role.

Please see implementation in other screens and do the same here